ansible/configs/sap-integration/default_vars.yml

```diff
@@ -0,0 +1,384 @@
+# Some valid: development, ilt, production, event
+purpose: development
+
+# Enable this if you want to use IPA for user authentication.
+# Mutually exclusive with {{ install_student_user }}
+install_ipa_client: false
+
+# Enable this if you want to create a user on the bastion
+# Mutually exclusive with {{ install_ipa_client }}
+install_student_user: false
+
+# TODO: What does this really do besides run the role?
+set_env_authorized_key: true
+env_authorized_key: "{{guid}}key"
+key_name: "default_key_name"
+
+# Run the bastion-lite role
+install_bastion: true
+
+# This config was written with the expectation of using python3
+# Several of the roles that it depends on only work with python2,
+# so they had to be modified to handle python3 as well. Use this
+# var so that it is passed along and influences how those roles run.
+all_use_python3: true
+
+# FTL is used for grading and solving. It will pull in the external ftl-injector role.
+# This might be enabled when we have solvers to run or graders for ILT
+install_ftl: false
+
+# TODO: Decide on whether to use sat or give access to repos directly with key
+# This will tell Agnosticd to use either:
+# satellite, rhn, or file for repos
+# Only satellite is supported for RHEL 8
+repo_method: satellite
+
+# If using satellite, these are needed and should
+# come from secrets:
+# satellite_url: satellite.opentlc.com
+# satellite_org: # This should be stored in secrets
+# satellite_activationkey: # This should be stored in secrets
+
+# use_content_view: true
+# If using file, these are needed in addition to the repos_template.j2 file:
+osrelease: '4.4.0'
+repo_version: '4.4'
+
+# own_repo_path: points to a repo mirror. Must defined in secrets
+#own_repo_path: <FROM_SECRETS>
+
+# Packages to install on all of the hosts deployed as part of the agnosticd config
+# This invokes the "common" role
+install_common: true
+
+# As part of the "common" role, this cause it to do a yum update on the host
+update_packages: true
+
+# Number of Nodes
+master_instance_count: 3
+worker_instance_count: 3
+
+# Install OpenShift 4 - and which version
+install_ocp4: true
+ocp4_installer_version: "4.4.9"
+
+# Run logic to enable cluster shutdown before 24h initial certificate rotation 
+# Only works for OCP 4.1 and 4.2. OCP 4.4 should no longer require this.
+ocp4_enable_cluster_shutdown: false
+
+# Install an OpenShift 4 Dev Preview Release
+# ocp4_installer_use_dev_preview: true
+# ocp4_installer_version: 4.4.0
+# ocp4_installer_url: https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp-dev-preview/latest-4.4/openshift-install-linux.tar.gz
+# ocp4_client_url: https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp-dev-preview/latest-4.4/openshift-client-linux.tar.gz
+
+# Pull secret needs to be defined in secrets
+# Get from https://try.openshift.com
+# ocp4_pull_secret: ''
+
+# Run smoketests after installation
+run_smoke_tests: false
+
+# Run fio performance tests at the end
```

Comment:
These should not be required - they were for specific configs.

Reply:
done